libr/arch/p/m680x_cs/plugin.c

```diff
@@ -515,6 +515,14 @@ static bool decode(RArchSession *as, RAnalOp *op, RAnalOpMask mask) {
 					insn->op_str[0]?" ": "", insn->op_str);
 			r_str_replace_in (op->mnemonic, strlen (op->mnemonic) + 1,
 				"ptr ", "", true);
+			// we don't really output operands in intel order but we can clean up
+			if (as->config->syntax == R_ARCH_SYNTAX_INTEL) {
```

Comment:
uhm, is this $ an immediate and # a load on the given address? because if its a load we can use [addr] instead

Reply:
Oh it's actually just like for 6502, # is immediate, and $ is for hex, which is the easy part.

I could go like replace all the "lda " and other insns that accept immediate with "lda [", then replace "[#" by "", then append the missing ] to those lines that still have the [ …

Reply:
yep, that may work

Reply:
Still incomplete but seems to work for now.